docs/guides/custom-extensions.md

```diff
@@ -42,6 +42,60 @@ class MyExtension(Extension):
         print('GraphQL request end')
 ```
 
+<details>
+  <summary>Extend error response format</summary>
+
+```python
+import strawberry
+from graphql import GraphQLError
```

Comment:
I think it might be worth having our own `GraphQLError` extension, which can extend from `graphql.GraphQLError`

I think it might be a better experience to expose GraphQLError from the strawberry namespace and it will also be useful for the future (in case we migrate to something else)

Would you be willing of adding this exception? No worries if you don't have time :)

Reply:
@patrick91 thanks for Great product and the review. I will give it a try.